index.md

```diff
@@ -1,26 +1,15 @@
-Anbox Cloud provides a rich software stack that enables you to run Android applications in the cloud for all kinds of different use cases, including high-performance streaming of graphics to desktop and mobile client devices.
+Anbox Cloud is a cloud platform that containerises mobile workloads using Android as a guest operating system.
 
-Using container technology, Anbox Cloud is scalable from a single-machine installation that can run scores of single Android systems to an entire cluster of machines. It is based on powerful and battle-proven software from Canonical like [LXD](https://linuxcontainers.org/) and [Juju](https://jujucharms.com/).
+It provides a rich software stack that enables you to run Android applications in the cloud. Anbox Cloud manages the container and application life cycle while optimising for high density, performance and fast container boot times.
```

Comment:
I wonder if one of these first two paragraphs should mention the relationship of Anbox Cloud to Anbox. Even a "...using open-source [Anbox](https://anbox.io) containerisation technology." at the end of one of the existing sentences would provide some useful orientation.